generators/config/templates/capybara.rb

```diff
@@ -158,6 +159,22 @@ module CapybaraHelpers
   Capybara::Selenium::Driver.new(app, browser: :remote, url: Howitzer.selenium_hub_url, desired_capabilities: caps)
 end
 
+# :crossbrowsertesting driver
+
+Capybara.register_driver :crossbrowsertesting do |app|
+  cap = {}
+  cap['name'] = Howitzer.cloud_cbt_name
+  cap['build'] = Howitzer.cloud_cbt_build
+  cap['browser_api_name'] = Howitzer.cloud_browser_name + Howitzer.cloud_browser_version
+  cap['os_api_name'] = Howitzer.cloud_platform
+  cap['screen_resolution'] = Howitzer.cloud_cbt_screen_resolution
+  cap['record_video'] = Howitzer.cloud_cbt_record_video
+  cap['record_network'] = Howitzer.cloud_cbt_record_network
```

Comment:
Please add `max_duration` option to capabilities.
https://help.crossbrowsertesting.com/selenium-testing/tutorials/crossbrowsertesting-automation-capabilities/#default